File: 01-Ejercicios-Clases-2023_2/C8-09-01-23--09-08-23/ProyectosJAVA/LP2SoftRRHHModel/src/pe/edu/pucp/lp2soft/rrhh/model/CuentaUsuario.java
package pe.edu.pucp.lp2soft.rrhh.model;
public class CuentaUsuario {
    private int idCuentaUsuario;
    private String username;
    private String password;
    private boolean activo;
    private Empleado empleado;

    public int getIdCuentaUsuario() {
        return idCuentaUsuario;
    }

    public void setIdCuentaUsuario(int idCuentaUsuario) {
        this.idCuentaUsuario = idCuentaUsuario;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }

    public Empleado getEmpleado() {
        return empleado;
    }

    public void setEmpleado(Empleado empleado) {
        this.empleado = empleado;
    }
    
}

File: 04-Codigo-Clases/BackCSharp_FrontJava/CSharp/LP2Soft/LP2SoftRRHHModel/CuentaUsuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2SoftRRHHModel
{
    public class CuentaUsuario
    {
        private int _idCuentaUsuario;
        private string _username;
        private string _password;
        private bool _activo;

        public int IdCuentaUsuario { get => _idCuentaUsuario; set => _idCuentaUsuario = value; }
        public string Username { get => _username; set => _username = value; }
        public string Password { get => _password; set => _password = value; }
        public bool Activo { get => _activo; set => _activo = value; }
    }
}
